packages/material-ui/src/Stepper/Stepper.js

```diff
@@ -56,14 +56,15 @@ function Stepper(props) {
   const childrenArray = React.Children.toArray(children);
   const steps = childrenArray.map((step, index) => {
     const controlProps = {
-      index,
       orientation,
       last: index + 1 === childrenArray.length,
```

Comment:
That should be removed now. It's causing prop-type warnings because it's spread to the div of the Step.